wiki/Tournaments/TWC/2020/en.md

```diff
@@ -0,0 +1,225 @@
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2020-02-06 & 2020-02-15 |
+| Live Drawings | 2020-02-29 (14:00 UTC) |
+| Group Stage | 2020-03-07 & 2020-03-08 |
+| Round of 16 | 2020-03-14 & 2020-03-15 |
+| Quarterfinals | 2020-03-21 & 2020-03-22 |
+| Semifinals | 2020-03-28 & 2020-03-29 |
+| Finals | 2020-04-04 & 2020-04-05 |
+| Grand Finals | 2020-04-11 & 2020-04-12 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | $150 per team member, unique profile badge, "osu!taiko Champion" user title for one year |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | $80 per team member, unique profile badge |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | $40 per team member, unique profile badge |
+
+## Organisation
+
+The osu!taiko World Cup 2020 is an upcoming event run by various community members, by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656), ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) |
+| Map Selector | ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766), ![][flag_JP] [tasuke912](https://osu.ppy.sh/users/2774767) |
+| Commentator | To be Announced...
+| Referee | ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Statistician | ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370), ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1018778)
+- [Livestream](https://www.twitch.tv/osulive)
+- **Statistics sheet** will be available shortly after the Live Drawings on Feb 29th (14:00 UTC)
+
+---
+
+## Participants
+
+Participating countries wil be anounced once the Registration phase ends and all submitted registrations have been verified.
+Participating players will be announced once confirmed by their respective country's elected team captain.
+
+---
+
+## Mappools
+
+Mappools will be announced one week prior to the commencement of each stage of the tournament.
+
+---
+
+## Ruleset
+
+### Tournament Rules
+
+1. The osu!taiko World Cup is a country-based team tournament, played on the osu!taiko game mode.
+   - **While this competition is planned as a 2 versus 2 setup, this might change depending on the amount of incoming registrations.**
+2. Beatmap scoring is based on Score V2.
+3. The maps for each round will be announced by the mapset selector in advance on the Sunday before the actual matches take place. Only these will be used during the respective matches.
```

Comment:
clarifying 'these' might be good

> Only these maps will be used during the respective matches.

Reply:
I don't think this really adds much personally